### DISCUSSION TITLE
Checkout immutable change

### DISCUSSION THREAD

**Discussion by kriomant** [jj-vcs/jj#5156] [Q&A] [answered]
Sometimes I want to checkout, e.g. root of current branch. Say, in order to check that some things which are broken now aren't broken there and it's my fault.

`jj edit` says "Error: Commit … is immutable"

What is the proper way to handle it?
 * use `jj new -A…`? but I'll then forget to drop it,
 * use `--ignore-immutable`. But I don't want to accidentally change root commit,
 * use `git checkout`
 * anything else?

**Comment by martinvonz** [comment, marked as answer]
> * use `jj new -A…`?

Without the `-A`. 

> but I'll then forget to drop it,

It should be dropped automatically when you update away from it.
